app/controllers/sections_controller.rb

```diff
@@ -101,6 +101,20 @@ def update_tags
     end
   end
 
+  def update_pdv
+    if @section.section_question_ids.include?(params[:sq_id])
+      sq = SectionQuestion.find(params[:sq_id])
+      sq.program_var = params[:pdv]
+      if sq.save!
+        render :show, status: :ok, location: @survey
+      else
+        render json: @section.errors, status: :unprocessable_entity
+      end
+    else
+      render json: @section.errors, status: :unauthorized
```

Comment:
Minor nit - Seems like an odd response code for the situation, maybe `:unprocessable_entity` here too?

Reply:
My original thought here was to try and account for the case someone tries to update a PDV on a SQ they don't own. Since load_and_authorize is only checking if they have authorization on the Section, they could in theory pass a section they have edit or group rights but a SQ id they don't and thus they should get denied. :unprocessable_entity does make more sense in the general case that it is checking though, an incorrect sqId on accident, so I'll make that change.